ViewTemplates/Passkeysmfa/error.blade.php

```diff
@@ -8,13 +8,11 @@
 defined('AKEEBA') || die;
 
 ?>
-<div id="passkeys-missing" style="margin: 0.5em 0">
-	<div class="alert alert-danger">
-		<h4>
-			@lang('PANOPTICON_MFA_PASSKEYS_ERR_NOTAVAILABLE_HEAD')
-		</h4>
-		<p>
-			@lang('PANOPTICON_MFA_PASSKEYS_ERR_NOTAVAILABLE_BODY')
-		</p>
-	</div>
-</div>
+<div class="alert alert-danger my-3 d-none" id="passkeys-missing">
+    <h3 class="alert-heading">
```

Comment:
Add an `h4` class here. The heading size was chosen based on what looks better in mobile devices with rather stout screens (I used an iPhone SE 2020).

Reply:
it was changed to exactly match the heading used in `ViewTemplates\Main\heartbeat.blade.php` and all other alert headings. This was the exception - but I can change it back if you insist

Reply:
I would rather it be a smaller heading in this particular case.

Reply:
done